### PR TITLE
1.7.9 hotfix

### PR DESCRIPTION
**Pull request by Umutayb**

### CODE REVIEW

File: src/main/java/pickleib/utilities/element/ElementInteractions.java [hunk 1]
```diff
@@ -0,0 +1,605 @@
...
+     * @param elementName target element name
+     * @param pageName specified page instance name
+     */
+    public void center(WebElement element, String elementName, String pageName){
+        log.info("Centering " +
+                highlighted(BLUE, elementName) +
+                highlighted(GRAY," on ") +
+                highlighted(BLUE, pageName)
+        );
+        center(element);
```
Comment (egecansen) [inline src/main/java/pickleib/utilities/element/ElementInteractions.java:140]: double logging?

File: src/main/java/pickleib/utilities/element/ElementInteractions.java [hunk 2]
```diff
@@ -0,0 +1,602 @@
...
+     * @param elementName target element name
+     * @param pageName specified page instance name
+     */
+    public void clickTowards(WebElement element, String elementName, String pageName){
+        log.info("Clicking " +
+                highlighted(BLUE, elementName) +
+                highlighted(GRAY," on the ") +
+                highlighted(BLUE, pageName)
+        );
+        clickAtAnOffset(element, 0, 0);
```
Comment (egecansen) [inline src/main/java/pickleib/utilities/element/ElementInteractions.java:168]: double logging?

File: src/main/java/pickleib/utilities/element/ElementInteractions.java [hunk 3]
```diff
@@ -0,0 +1,602 @@
...
+     * @param pageName specified page instance name
+     */
+    public void clickIfPresent(WebElement element, String elementName, String pageName){
+        log.info("Clicking " +
+                highlighted(BLUE, elementName) +
+                highlighted(GRAY," on the ") +
+                highlighted(BLUE, pageName) +
+                highlighted(GRAY, ", if it is present...")
+        );
+        clickButtonIfPresent(element);
```
Comment (egecansen) [inline src/main/java/pickleib/utilities/element/ElementInteractions.java:186]: log? @Umutayb

**Comment by Umutayb** [comment]
@egecansen what do you think?

File: src/main/java/pickleib/utilities/element/ElementInteractions.java [hunk 4]
```diff
@@ -0,0 +1,602 @@
...
+     * @param pageName specified page instance name
+     */
+    public void verifyPresence(WebElement element, String elementName, String pageName){
+        log.info("Verifying presence of " +
+                highlighted(BLUE, elementName) +
+                highlighted(GRAY," on the ") +
+                highlighted(BLUE, pageName)
+        );
+        verifyElementState(element, ElementState.displayed);
+        log.success("Presence of the element " + elementName + " was verified!");
```
Comment (egecansen) [inline src/main/java/pickleib/utilities/element/ElementInteractions.java:289]: is this double logging? @Umutayb

File: src/main/java/pickleib/utilities/element/ElementInteractions.java (hunk 2 above)
Comment (egecansen) [reply src/main/java/pickleib/utilities/element/ElementInteractions.java:168]: @Umutayb

File: src/main/java/pickleib/utilities/element/ElementInteractions.java (hunk 1 above)
Comment (egecansen) [reply src/main/java/pickleib/utilities/element/ElementInteractions.java:140]: @Umutayb

File: src/main/java/pickleib/utilities/element/ElementInteractions.java [hunk 5]
```diff
@@ -0,0 +1,605 @@
...
+     * @param buttonName target button name
+     * @param pageName specified page instance name
+     */
+    public void clickInteraction(WebElement button, String buttonName, String pageName){
+        log.info("Clicking " +
+                highlighted(BLUE, buttonName) +
+                highlighted(GRAY," on the ") +
+                highlighted(BLUE, pageName)
+        );
+        clickElement(button);
```
[the remark this answers is not in the document]
Comment (egecansen) [reply src/main/java/pickleib/utilities/element/ElementInteractions.java:96]: @Umutayb

**Pull request merged**